tests/unit/test_agent_pricing_exceptions.py

```
@@ -0,0 +1,78 @@
+
+
+class _StubCore:
+    def __init__(self) -> None:
+        self._agent_runner = _StubAgentRunner()
+        self._processor_pipeline = _StubProcessorPipeline()
+        self._usage_meter = _StubUsageMeter()
+
+    def _safe_step_name(self, step):
+        try:
+            return step.name
+        except Exception:
```

Ruff (0.14.7)

Do not catch blind exception: `Exception`

(BLE001)